### ISSUE TITLE
Meta Title missing on pages

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#1348]
Every page needs a meta title in the head. You can usually just use the text in the H1 as the title. Attached are a few examples like /myforms.

Requirement: [2.4.2 Page Titled (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.2)

Note: /form-builder, /form-builder/edit and related do have a title

[Screenshot 2022-11-29 at 11.32.13 AM.png](https://images.zenhubusercontent.com/63321fcd2e391c72611f2316/9acac5f8-0c74-46f8-bfb7-4684acfb16ff)

![Screenshot 2022-11-29 at 12.00.48 PM.png](https://images.zenhubusercontent.com/63321fcd2e391c72611f2316/6f595405-71fa-4b30-b517-15234e5bb675)

![Screenshot 2022-11-29 at 12.01.04 PM.png](https://images.zenhubusercontent.com/63321fcd2e391c72611f2316/f3d7f51a-966d-4dae-b16c-5a31cb1830e6)

![Screenshot 2022-11-29 at 12.01.22 PM.png](https://images.zenhubusercontent.com/63321fcd2e391c72611f2316/01d0b84b-85e3-4d14-9112-aa6503ba3f7c)
